src/main/java/org/broadinstitute/hellbender/utils/downsampling/ReadsDownsampler.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.hellbender.utils.downsampling;
+
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * An extension of the basic downsampler API with reads-specific operations
+ */
+public abstract class ReadsDownsampler<T extends GATKRead> extends Downsampler<T> {
```

Comment:
Can you refactor to make this `ReadsDownsampler extends Downsampler<GATKRead>`? The only reason this had a type parameter was to allow strongly-typed operation on both `SAMRecord` and `GATKSAMRecord` in GATK3 -- now that we have a unified reads interface in GATK4, this is no longer necessary.